core/src/main/java/org/apache/calcite/sql2rel/RelFieldTrimmer.java

```diff
@@ -1072,6 +1074,26 @@ public TrimResult trimFields(
       ++j;
     }
 
+    if (newAggCallList.isEmpty() && newGroupSet.isEmpty()) {
```

Comment:
Maybe an empty value rel should be used here instead

Reply:
Done.